corehq/apps/app_manager/templates/app_manager/v1/partials/app-settings.html

```diff
@@ -172,6 +172,14 @@
 "></inline-edit>
 
 <div class="tab-pane" id="commcare-settings">
+    {% if app.get_doc_type == 'LinkedApplication' %}
+        <p> Your app is at version {{ app.version }}. The master app is at version {{ master_version }}. </p>
```

Comment:
Since its in a different domain, there are possibly confusing permission issues to users. Particularly since we tend to use a 404 to mean you are not allowed to see something. Having a link that 404s for most of the web users seemed more trouble than it was worth but it does seem nice to have that viewable somewhere.